src/Event/EventManager.php

```diff
@@ -442,6 +442,24 @@ public function prioritisedListeners($eventKey)
     }
 
     /**
+     * Returns the listeners matching a specified pattern
+     *
+     * @param string $eventKeyPattern Pattern to match.
+     * @return array
+     */
+    public function matchingListeners($eventKeyPattern)
+    {
+        $matchPattern = '/' . preg_quote($eventKeyPattern, "/") . '/';
```

Comment:
No worries Mark. I know you have a life and career outside CakePHP work. I'll get things patched up and tweak the unit-tests accordingly and we'll go from there. :+1: